assets/lang/sv-SE.json

```
@@ -890,5 +890,33 @@
       "choice-pfnts": false,
       "choice-passed-away": false
     }
+  },
+
+  "diabetes": {
+    "which-type": "Du angav tidigare att du har diabetes, vilket är en viktig riskfaktor för covid-19. Våra forskare har några uppföljande frågor för att de ska förstå mer om detta.",
```

this is the incorrect string, you copied the justification by mistake - it should be 
```suggestion
    "which-type": "Vilken typ av diabetes har du?",
```